Projects/UOContent/Engines/Spawners/EditSpawnCommand.cs

```diff
@@ -67,45 +68,65 @@ public override void ExecuteList(CommandEventArgs e, List<object> list)
 
             var argSpan = e.ArgString.AsSpan(name.Length + 1);
             var setIndex = argSpan.InsensitiveIndexOf("set ");
+            var where = argSpan.InsensitiveIndexOf("match ");
+           
+            ReadOnlySpan<char> props = null, findmatch = null;
 
-            ReadOnlySpan<char> props = null;
-
-            if (setIndex > -1)
+            if (setIndex > -1 || where > -1)
             {
                 var start = setIndex + 4;
-                props = argSpan.Slice(start, argSpan.Length - start);
-                argSpan = argSpan.SliceToLength(setIndex);
+                var len = where>-1? where : argSpan.Length;
+                findmatch = argSpan.Slice(len < argSpan.Length ? len + 6 : argSpan.Length);
+                if (setIndex>-1)
+                {
+                    props = argSpan.Slice(start, len - start);
+                    argSpan = argSpan.SliceToLength(setIndex);
+                }
+                else
+                {
+                    argSpan = argSpan.SliceToLength(argSpan.Length-len);
+                }
             }
 
             var argStr = argSpan.ToString().DefaultIfNullOrEmpty(null);
             var propsStr = props.ToString().DefaultIfNullOrEmpty(null);
+            var whereStr = findmatch.ToString().DefaultIfNullOrEmpty(null);
+            //this is fix for search like  match str 10  if mob had str 100 it's find it
+            if (whereStr != null) (whereStr += " ").ToLower();
```

Comment:
This doesn't actually do anything 😆. ToLower() returns a lowercase string, but you aren't using it.

Reply:
I can put to console param like  match STR  ;d